.github/workflows/ci.yml

```
@@ -322,28 +334,69 @@ jobs:
+  #     env:
+  #       CONTACT_VAR: ${{ vars.CONTACT }}
+  #   - name: Dependencies
+  #     run: |
+  #       sudo apt-get update
+  #   - name: Run test
+  #     working-directory: ./scala/coding
+  #     run: |
+  #       ls -all
+  #       sbt test
+  
+  test-scala-coding:  
```

yamllint

[error] trailing spaces

(trailing-spaces)